### PR TITLE
Skip the on-heap trampoline for builtin calls

### PR DESCRIPTION
**Pull request by luyahan**
If using on-heao trampoline, d8 running jitless mode will be crash.
Refer to https://chromium-review.googlesource.com/c/v8/v8/+/1382461

### CODE REVIEW

File: BUILD.gn [hunk 1]
```diff
@@ -4179,6 +4179,10 @@ v8_component("v8_libbase") {
     sources += [ "src/base/ubsan.cc" ]
   }
 
+  if(v8_current_cpu == "riscv" || v8_current_cpu == "riscv64"){
+    libs += [ "atomic" ]
+  }
+
```
Comment (penguinwu) [inline BUILD.gn:4185]: How is this related to skip online trampoline?

Comment (luyahan) [reply BUILD.gn:4185]: Building native neeads it

**Review by penguinwu — APPROVED** [review]
LGTM
